gum/gumexceptor.h

```diff
@@ -24,7 +24,9 @@ G_DECLARE_FINAL_TYPE (GumExceptor, gum_exceptor, GUM, EXCEPTOR, GObject)
 #else
 # define GUM_NATIVE_SETJMP(env) sigsetjmp (env, TRUE)
 # define GUM_NATIVE_LONGJMP siglongjmp
+# if defined (HAVE_FRIDA_GLIB)
```

Comment:
It's valid to be using a shared build of Frida's GLib with GIR so this is the wrong check – can we get GIR to ignore it instead?

Reply:
Using GLib annotations (like you saw for Capstone) just didn't work every single way I twisted it. I will try again, but yeah we need to figure out a better wayt to fix this (GObject Introspection generates an invalid GIR for this type, and I do not know of any other way to fix it... other than to ignore it)

Reply:
Oh, interesting. Perhaps GIR sets a preprocessor define that we can detect?